specs/DotNetProjectFile.Analyzers.Specs/TestTools/CachedProjectLoader.cs

```diff
@@ -11,6 +11,7 @@ internal static class CachedProjectLoader
 
     private sealed class Entry(FileInfo file)
     {
+        private readonly Random rnd = new(file.GetHashCode());
```

Comment:
May be document what is going on here. I assume the magic is needed, but I do not know why.